lib/dom.dart

```diff
@@ -453,24 +452,24 @@ class Text extends Node {
   @override
   String get text => data;
   @override
-  set text(String value) {
+  set text(String? value) {
```

Comment:
This is a little tougher.

We can't narrow the type from `Node.text=` which is nullable. If we make the setter non-nullable in that class it would clash with the getter which must be nullable, because the spec tells us that it must sometimes return null.

https://dom.spec.whatwg.org/#dom-node-textcontent

I could maybe go with `covariant` here?

Reply:
So the superclass has a mutable nullable property, and the subclass has a mutable non-nullable property?
That sounds unsound (that's DOM for you), and I'd probably try to see if `covariant` can work.

Reply:
Done